baywatch.install

```diff
@@ -8,100 +8,49 @@
 use Drupal\Core\Serialization\Yaml;
 use Drupal\Core\File\FileSystemInterface;
 
+use Drupal\baywatch\BaywatchOperationTrait;
+
 /**
  * Implements hook_install().
  */
 function baywatch_install() {
   if (\Drupal::service('module_handler')->moduleExists('bay')) {
-    \Drupal::service('module_installer')->uninstall('bay');
-  }
+        \Drupal::service('module_installer')->uninstall('bay');
+    }
 
-  // This Will import password policy during first install of the module.
-  baywatch_import_sdpa_password_policy();
+    // This Will import password policy during first install of the module.
+    BaywatchOperationTrait::import_sdpa_password_policy();
 
-  // Ensure queue_mail is configured to handle all mail.
-  baywatch_update_8002();
+    // Ensure queue_mail is configured to handle all mail.
+    BaywatchOperationTrait::enable_queue_mail();
 
-  // Import key.key.authenticated_content.
-  baywatch_update_8003();
+    // Import key.key.authenticated_content.
+    BaywatchOperationTrait::cleanup_tables();
 
-  // Import authenticated_content key.
-  baywatch_update_8004();
+    // Import authenticated_content key.
+    BaywatchOperationTrait::import_authenticated_content_key();
 
-  // Change config_split settings.
-  baywatch_update_8005();
+    // Change config_split settings.
+    BaywatchOperationTrait::enable_config_split();
 
-  // Enable the new preview module.
-  baywatch_update_8006();
+    // Enable the new preview module.
+    BaywatchOperationTrait::enable_preview();
 
-  // Enable the new share links module.
-  baywatch_update_8007();
+    // Enable the new share links module.
+    BaywatchOperationTrait::enable_share_links();
 
-  // Revoke the previewer role.
-  baywatch_update_8008();
-}
+    // Remove the Previewer role from users.
+    BaywatchOperationTrait::remove_previewer_role();
```

Comment:
I know you addressed this in a standup but I don't these functions are concerns of Baywatch. It's purpose is infrastructure/third-party service related and these appear more application related. Can they be pushed out to Tide projects?